Source/WebCore/Modules/webauthn/fido/Pin.cpp

```diff
@@ -69,6 +69,32 @@ static bool hasAtLeastFourCodepoints(const String& pin)
     return pin.length() >= 4;
 }
 
+static Vector<uint8_t> decryptForProtocol(PINUVAuthProtocol protocol, const CryptoKeyAES& key, const Vector<uint8_t>& ciphertext)
+{
+    if (protocol == PINUVAuthProtocol::kPinProtocol2) {
+        // CTAP 2.1 spec 6.5.7: Protocol 2 decrypt
+        // Split ciphertext into IV (first 16 bytes) and ct (remaining bytes)
+        if (ciphertext.size() < 16)
```

Comment:
Nit: Might be cleaner to make a constant `ivLength = 16` or something in case this length is ever changed in the spec.

It could be use in lines 80-81, too.